index.html

```diff
@@ -270,6 +270,35 @@ <h1 class="header" id="registration">Registration</h1>
       </div>
     </div>
 
+    <div class="tracks section" id="tracks">
+      <div class="container">
+        <h1 class="header">Tracks</h1>
+        <p>We are offering four different tracks for Technica this year! Each track is designed to best accommodate a hacker's interests and skill level with track-specific mentors. Hackers may express interest in multiple tracks during registration but will only be able to participate in one.</p>
+        <div class="text-center gallery" data-flickity='{ "contain": true, "wrapAround": true }'>
+          <div class="general track-flick-item">
+            <img class="track-image" src="./img/CottageHomeIcon.svg" alt="simple cottage home sketch">
+            <h3 class="track-title">GENERAL</h3>
+            <p class="track-text">For any and all hackers! Come build to your heart's content using hardware, software, or anything you want with other hackers! Hackers can be students, designers, or just any tech-lover out there. Hackers of all skill levels are welcomed and supported!</p>
+          </div>
+          <div class="research track-flick-item">
+            <img class="track-image" src="./img/LemonadeIcon.svg" alt="simple lemonade in pitcher and two glasses sketch">
+            <h3 class="track-title">RESEARCH</h3>
+            <p class="track-text">Calling all undergraduates interested in research! Come explore the fields of research within computer science. This track is a separate 3-day event taking place during Technica weekend where you’ll get to work hands-on with faculty researchers and peers to address social issues using technology and research!</p>
+          </div>
+          <div class="beginner track-flick-item">
+            <img class="track-image" src="./img/BikeIcon.svg" alt="simple bike with flower basket sketch">
+            <h3 class="track-title">BEGINNERS</h3>
+            <p class="track-text">New to hackathons? Don't know how to code? No worries! You'll be provided with dedicated resources to help you be successful at Technica and beyond! This track begins with some exclusive workshops a few weeks before Technica and includes everything from an introduction to hackathons to universal coding and logic concepts!</p>
+          </div>
+          <div class="hardware track-flick-item">
+            <img class="track-image" src="./img/FlowerBasketIcon.svg" alt="simple flower basket sketch">
```

Comment:
suuuper nitpicky but for consistency sake in img naming, can u name these to match the style of other files in the img folder (e.g. cottage-icon, lemondae-icon, bike-icon, flower-basket-icon)